tests/test_suite.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import argparse
+import logging
+import os
+import os.path
+import sys
+import time
+
+from ssg_test_suite.log import LogHelper
+import ssg_test_suite.oscap
+import ssg_test_suite.virt
+import ssg_test_suite.profile
+import ssg_test_suite.rule
+
+
+parser = argparse.ArgumentParser()
+
+common_parser = argparse.ArgumentParser(add_help=False)
+common_parser.add_argument("--hypervisor",
+                           dest="hypervisor",
+                           metavar="HYPERVISOR",
+                           default="qemu:///session",
+                           help="libvirt hypervisor")
+common_parser.add_argument("--domain",
+                           dest="domain_name",
+                           metavar="DOMAIN",
+                           required=True,
+                           help=("Specify libvirt domain to be used as a test "
+                                 "bed. This domain will get remediations "
+                                 "applied in it, possibly making system unusable "
+                                 "for a moment. Snapshot will be reverted "
+                                 "immediately afterwards. "
+                                 "Domain will be returned without changes"))
+common_parser.add_argument("--datastream",
+                           dest="datastream",
+                           metavar="DATASTREAM",
+                           default=("/usr/share/xml/scap/ssg/content/"
```

Comment:
Good call - it was convenient to have it this way during development, but makes no sense. Fixed.